modules/ROOT/pages/metrics/custom-metrics-connector.adoc

```diff
@@ -0,0 +1,96 @@
+= Anypoint Custom Metrics Connector
+
```

Comment:
Should remove this blank line after the title. All other docs I've seen don't have blank lines after a title.